docs/faq/differences-with.../singer-vs-airbyte.md

```diff
@@ -0,0 +1,24 @@
+# Singer vs Airbyte
+
+We wrote an article about this topic: “[Airbyte vs. Singer: Why Airbyte is not built on top of Singer](https://airbyte.io/articles/data-engineering-thoughts/airbyte-vs-singer-why-airbyte-is-not-built-on-top-of-singer/).” As a summary, here are the differences. 
+
+### **Singer:**
+
+* **Supports 96 connectors after 4 years.**
+* **Increasingly outdated connectors:** Talend \(acquirer of StitchData\) seems to have stopped investing in maintaining Singer’s community and connectors. As most connectors see schema changes several times a year, more and more Singer’s taps and targets are not actively maintained and are becoming outdated. 
+* **Absence of standardization:** each connector is its own open-source project. So you never know the quality of a tap or target until you have actually used it. There is no guarantee whatsoever about what you’ll get.
+* **Singer’s connectors are standalone binaries:** you still need to build everything around to make them work. 
```

Comment:
i know what you mean here, but i don't think it's obvious from the outside.
```suggestion
* **Singer’s connectors are standalone binaries:** you still need to build everything around to make them work (e.g. UI, configuration validation, state management, normalization, schema migration, monitoring, etc). 
```

Reply:
you don't need to include everything from that list. just a suggestion. maybe just pick the things you think people care about most.